pkg/asset/agent/manifests/infraenv.go

```diff
@@ -81,6 +81,9 @@ func (i *InfraEnv) Generate(dependencies asset.Parents) error {
 		if installConfig.Config.Proxy != nil {
 			infraEnv.Spec.Proxy = getProxy(installConfig)
 		}
+		if installConfig.Config.AdditionalTrustBundle != "" {
+			infraEnv.Spec.AdditionalTrustBundle = installConfig.Config.AdditionalTrustBundle
```

Comment:
> I reckon you should call getAdditionalTrustBundle(...) in pkg/asset/agent/manifests/common.go to obtain this (see line 82 for something similar) , I know it's only a one liner but at least it would give that code some purpose other than to simply service a test

Good idea, thanks @paul-maidment. Updated.